sunpy/tests/helpers.py

```diff
@@ -77,16 +75,27 @@ def test_simple_plot():
     def wrapper(*args, **kwargs):
         if not os.path.exists(hash.HASH_LIBRARY_FILE):
             pytest.xfail('Could not find a figure hash library at {}'.format(hash.HASH_LIBRARY_FILE))
+
+        name = "{0}.{1}".format(test_function.__module__,
+                                test_function.__name__)
         plt.figure()
-        name = "{0}.{1}".format(test_function.__module__, test_function.__name__)
         pngfile = tempfile.NamedTemporaryFile(delete=False)
         figure_hash = hash.hash_figure(test_function(*args, **kwargs), out_stream=pngfile)
         figure_test_pngfiles[name] = pngfile.name
         pngfile.close()
+
+        # Save the image that was generated
+        os.makedirs('result_images', exist_ok=True)
```

Comment:
I wonder whether this could be saved into a directory like where the `sunpy-figure-tests` repository would be.

Reply:
there should probably be a separate copy step, as this should also work on peoples computers etc.